doc/api/esm.md

```diff
@@ -586,12 +558,27 @@ an all-ES module-syntax version the package. This could be used via `import
 'pkg/module'` by users who are certain that the CommonJS version will not be
 loaded anywhere in the application, such as by dependencies; or if the CommonJS
 version can be loaded but doesn’t affect the ES module version (for example,
-because the package is stateless).
+because the package is stateless). This variant would not require
+`--experimental-conditional-exports`:
```

Comment:
I know you mention it here again, but since the first section covered this I don't think we need another code example.

Reply:
Sorry, no this is exactly the code example we need.